README.md

```diff
@@ -11,3 +11,20 @@ Kubernetes. This specific project will include:
 * Writing a report that summarizes the different approaches that can be used for
   topology aware routing along with a recommendation.
 
+## Usage
+go run main.go -input=inputFile -output=outputFile -alg=algorithm
+
+example of intput file (csv): each zone with number of nodes first, number of endpoints next  
+input name, zone1, zone2, zone3  
+perfect input, 10 10, 10 10, 20 20
```

Comment:
The example input files I've seen don't have spaces after the commas, just wanted to confirm that this also works.

Reply:
yeah, this works as well.